homeassistant/components/netatmo/strings.json

```
@@ -58,7 +58,9 @@
   "device_automation": {
     "trigger_subtype": {
       "away": "[%key:common::state::not_home%]",
+      "home_id": "Home Id",
       "schedule": "Schedule",
+      "schedule_id": "Schedule Id",
```

Abbreviations should be capitalized:

```suggestion
      "home_id": "Home ID",
      "schedule": "Schedule",
      "schedule_id": "Schedule ID",
```